R/add_nsu_cohort.R

```diff
@@ -24,7 +24,8 @@ add_nsu_cohort <- function(data, year) {
 
   matched <- dplyr::full_join(data,
     # NSU cohort file
-    haven::read_sav(get_nsu_path(year, ext = "zsav")),
+    haven::read_sav(get_nsu_path(year, ext = "zsav")) %>%
+      dplyr::mutate(gpprac = as.integer(.data$gpprac)),
```

Comment:
This will convert to an integer but also recode any English practices (starts with a letter) to the dummy code, rather than just `NA`
```suggestion
      dplyr::mutate(gpprac = convert_eng_gpprac_to_dummy(.data$gpprac)),
```